mountain-climbing/src/main/java/com/example/mountainclimbing/controller/ImageController.java

```diff
@@ -0,0 +1,53 @@
+package com.example.mountainclimbing.controller;
+
+import java.util.Optional;
+
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.mountainclimbing.model.Image;
+import com.example.mountainclimbing.service.ImageService;
+
+@RestController
+@RequestMapping("/image")
+public class ImageController {
+	
+	private final ImageService imageService;
+	
+	public ImageController(ImageService imageService) {
+		// TODO Auto-generated constructor stub
+		this.imageService = imageService;
+	}
+	
+	@PostMapping
+	@ResponseBody
+	public Image createImage(@RequestBody Image image) {
+		return imageService.createImage(image);
+	}
+	
+	@GetMapping
+	@ResponseBody
```

Comment:
Response body is not necessary because RestController annotation already have @ResponseBody.